lib/ansible/modules/windows/win_copy.py

```diff
@@ -91,6 +91,7 @@
       end with "/", the directory itself with all contents is copied.
     - If path is a file and dest ends with "\", the file is copied to the
       folder with the same filename.
+    - Not required if using C(content)
```

Comment:
remove `required: yes`, if this is the case, that is incorrect documentation, make note elsewhere that 'either content or src is required', but each one should be `required: no` themselves.